GUI/QtQuick/TxInputsModel.cpp

```diff
@@ -48,17 +51,27 @@ QVariant TxInputsModel::data(const QModelIndex& index, int role) const
       return getData(index.row(), index.column());
    case HeadingRole:
       return (index.row() == 0);
-   case WidthRole:
-      return colWidth(index.column());
+   case SelectedRole:
+      return (index.row() > 0 && index.column() == 0) ? (selection_.find(index.row() -1) != selection_.end()) : false;
+   case ExpandedRole:
+      return (index.row() > 0 && index.column() == 0) ? data_[index.row() - 1].expanded : false;
+   case CanBeExpandedRole:
+      return (index.row() > 0 && index.column() == 0) ? data_[index.row() - 1].txId.empty() : false;
    case ColorRole:
       return dataColor(index.row(), index.column());
-   case BgColorRole:
-      return bgColor(index.row());
    default: break;
    }
    return QVariant();
 }
 
+QColor TxInputsModel::dataColor(int row, int col) const
+{
+   if (row == 0) {
+      return QColor("#7A88B0");
+   }
+   return QColor("#FFFFFF");
```

Comment:
Shouldn't we use QML theme colors everywhere?

Reply:
In tables we use - ColorScheme (from ColorScheme.h). I agree that will be better use qml...but deadlines...so to now I suggest use ColorScheme, and in future we will do refactoring